storybook/src/components/Header/Header.docs.mdx

```diff
@@ -32,14 +32,14 @@ A Page Menu in the Header should not wrap.
 
 <Canvas of={HeaderStories.WithLinks} />
 
-## With links and menu
+## With app name and menu
```

Comment:
This doesn't have an app name?

Reply:
I blame Vincent use of webstorm ;)

Reply:
Unjustly so, you know I’m an excellent renamer 😎

It was actually a slip of your own keyboard: https://github.com/Amsterdam/design-system/commit/6e86af8c117cee786afd1896e35db79d19c1085b#diff-14fc32124a34d0ee7bc7ac3376a0aab2702a9e7dc359307662fab46821ddf114R35

Reply:
🤦‍♂️